service/src/test/resources/dummyPaymentOneAuthPending20Euro_PDT.json

```diff
@@ -0,0 +1,56 @@
+{
+  "id": "40a495a2-d709-4484-88cb-d4129acffda7",
+  "version": 1,
+  "amountPlanned": {
+    "currencyCode": "EUR",
+    "centAmount": 2000
+  },
+  "customer": {
+    "typeId": "customer",
+    "id": "276829bd-6fa3-450f-9e2a-9a8715a9a104",
+    "obj": {
+      "id": "276829bd-6fa3-450f-9e2a-9a8715a9a104",
+      "version": 1,
+      "dateOfBirth": "1989-12-03",
+      "customerNumber": "123"
+    }
+  },
+  "paymentMethodInfo": {
+    "paymentInterface": "PAYONE",
+    "method": "WALLET-PAYDIREKT"
+  },
+  "paymentStatus": {},
+  "transactions": [
+    {
+      "id": "58728a39-c2f5-4467-ab17-b329b55c4423",
+      "timestamp": "2015-12-03T10:00:31.498Z",
+      "type": "Authorization",
+      "amount": {
+        "currencyCode": "EUR",
+        "centAmount": 2000
+      },
+      "interactionId": "0",
+      "state": "Pending"
+    }
+  ],
+  "interfaceInteractions": [
+  ],
+  "interfaceId" : "0000123",
+  "custom": {
+    "type": {
+      "typeId": "string",
+      "id": "<type-id>"
+    },
+    "fields": {
+      "cardDataPlaceholder": "00023123451045243",
```

Comment:
I guess this in not required for paydirekt payment?

Reply:
9c200184